src/app/core/feature-module/featured.routes.ts

```diff
@@ -10,26 +10,22 @@ export const FEATURED_ROUTES = {
     return `${environment.apiURL}/featured/learning-objects`;
   },
   /**
-   * Request to retrieve the featured learning objects for a specific collection
+   * Request to retrieve the featured learning objects for a specific collection with an optional limit
    * @method GET
    * @param collectionAbvName the abbreviated name of the collection
+   * @param limit (optional) the number of featured learning objects to retrieve
    */
-  GET_COLLECTION_FEATURED_OBJECTS(collectionAbvName: string) {
+  GET_COLLECTION_FEATURED_OBJECTS(
+    collectionAbvName: string,
+    limit?: number,
+  ) {
+    if (typeof limit !== 'undefined') {
+      return `${environment.apiURL}/featured/learning-objects?collection=${encodeURIComponent(
+        collectionAbvName,
+      )}&limit=${encodeURIComponent(limit)}`;
+    }
     return `${environment.apiURL}/featured/learning-objects?collection=${encodeURIComponent(collectionAbvName)}`;
```

Comment:
I'd recommend doing this type of check. Using `if(<variable>)` will check if the variable exists. Also instead of creating two unique strings you can append the additional query to the first string.

```suggestion
    let url = `${environment.apiURL}/featured/learning-objects?collection=${encodeURIComponent(collectionAbvName)}`;
    if (limit) {
          url += `&limit=${encodeURIComponent(limit)}`;
    }
    return url;
```